crates/wit/src/lib.rs

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-#![doc(html_root_url = "https://docs.rs/wit-support/0.2.12")]
+#![doc(html_root_url = "https://docs.rs/wit-support/0.2.13")]
```

Comment:
I wonder if it is possible to use something like `concat!(env!(...))`
```suggestion
#![doc(html_root_url = concat!("https://docs.rs/wit-support/", env!("CARGO_PKG_VERSION"))]
```

Reply:
Great, thanks)